test/test_utils/TestMaterialAppWidget.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:empriusapp/src/core/routes.dart';
+import 'package:empriusapp/src/core/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'TestNavigationObserver.dart';
+
+class TestMaterialAppWidget extends StatelessWidget {
+  //final NavigatorObserver navigatorObserver;
+  final Widget home;
+
+  TestMaterialAppWidget({
+    //Key key,
+    required this.home
+    //this.navigatorObserver
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ProviderScope(
+      child: MaterialApp(
+        //title: 'Widget Test',
+        home: home,
+        //navigatorObservers: [navigatorObserver],
+        //routes: _testRoutes(),
+      ),
+    );
+  }
+}
+
+/*  Map<String, WidgetBuilder> _testRoutes() {
```

Comment:
Commented code MA-LO

Reply:
tests should not be here its a WIP branch indeed